backend/public-graph/graph/resolver.go

```diff
@@ -963,6 +964,7 @@ func GetLocationFromIP(ctx context.Context, ip string) (location *Location, err
 
 	location = &Location{
 		City:      record.City.Names["en"],
+		State:     record.Subdivisions[0].Names["en"],
```

Comment:
is `Subdivisions` guaranteed to be non-empty? should we have a check to avoid a panic here?

Reply:
Yeah, sorry, we actually have handling for it below, I just missed it while stepping through code on a breakpoint.

https://github.com/highlight/highlight/pull/9080/files#diff-37a818bb25fd3cee29d865b3306a81501a1371a5ba2e6f1927464f21a56b20a1L972-L974